client/rest-client/src/main/java/org/infinispan/client/rest/RestResponse.java

```diff
@@ -17,6 +17,11 @@
 
    Map<String, List<String>> headers();
 
+   /**
+    * Returns the value of a header as a String. For multi-valued headers, values are separated by comma.
+    */
+   String getHeader(String header);
```

Comment:
why a String and not a Collection/Array?

Reply:
Because 99% of the headers are single values...And there is another method for them